h2o-core/src/main/java/hex/grid/Grid.java

```diff
@@ -538,6 +535,31 @@ public void exportModelsBinary(final String exportDir) throws IOException {
     }
   }
 
+  /**
+   * Saves all of the frames used by this Grid's params. Frames are named by their keys.
+   * @param exportDir directory to export all the frames to
+   */
+  public void exportFramesBinary(final String exportDir) {
+    Map<String, Frame> frames = getParams().getAllFrames();
+    frames.values().stream()
+        .map(frame -> new FramePersist(frame).saveTo(exportDir, true))
```

Comment:
just a question: What file systems are supported by `FramePersist`? Are save and load operations distributed? Any who looks after cleaning `exportDir` when training is properly done?

Reply:
all that are supported by rest of H2O for a distributed cluster a distributed FS should be used
save and load are distributed, each node writes/reads only local chunks, hence thre requirement for same cluster size on load
cleaning of exportDir is left up to the user, this is the same behaviour with normal export_checkpoints_dir option

Reply:
Thanks for your explanation!